smart-hungry-delete.el

```diff
@@ -61,6 +61,12 @@
   :safe t
   )
 
+(defcustom smart-hungry-delete-add-kill-ring t
```

Comment:
Hmm, I don't want to be annoying, but can we rename this to `smart-hungry-delete-kill-ring-save` just like `kill-ring-save`? Or were you inspired by some other existing variable/function name? There's also `avy-kill-ring-save-region` and `org-export-copy-to-kill-ring`.